src/main/scala/example/Inheritance.scala

```diff
@@ -7,3 +7,10 @@ abstract class DocumentationInheritance[T, A <: Int, B >: String, -X, +Y] extend
 class DocumentationInheritanceMethod:
     def wierdMethod[T, A <: Int, B >: String](t: T, a: A): B = ???
     def threOtherWay[A <: Nothing, B >: Any](a: A, c: B): Unit = ???
+
+class A:
+    class B extends C:
+        class D extends C
```

Comment:
More tests would be good:
```suggestion
class A:
    type I = Int
    given Unit = ()
    extension (u: Unit) def foo = "foo"
    object X
    class B extends C:
        class D extends C
```

Reply:
changed